labs-ios-starter/Model/NetworkController.swift

```diff
@@ -17,19 +17,14 @@ protocol NetworkPlaceholder {
     func get(request: URLRequest, completionHandler: @escaping (Data?, URLResponse?, Error?) -> Void)
 }
 
-class NetworkController: GetUserInput {
+class NetworkController {
     
     var myUserInput: String?
     let network: NetworkPlaceholder
     init(network: NetworkPlaceholder = URLSession.shared) {
         self.network = network
     }
     
-    func capturingUserInput(userInput: String) {
-        myUserInput = userInput
-        print(myUserInput)
-    }
-    
```

Comment:
Thanks for fixing this. Forgot that I was conforming to this protocol that I removed.